lib/src/model.dart

```diff
@@ -180,6 +192,32 @@ class License {
   Map<String, dynamic> toJson() => _$LicenseToJson(this);
 }
 
+/// Describes a position in a source file.
+///
+/// WARNING: this class is experimental, do not rely on it.
+@JsonSerializable()
+class Position {
```

Comment:
Not sure, but would it make sense to make the class represent a range instead?

Reply:
Updated the code with `Range` (with `start` and `stop`). PTAL.

Note: maybe this is an overkill here, but we could also emit a diff format: while marking the recognized range, we could highlight added words and also identify removed words too. Just in case the license was modified in a non-trivial way.